spec/classes/jira_config_spec.rb

```diff
@@ -171,6 +172,15 @@
           it { should contain_file('/opt/jira/atlassian-jira-6.3.4a-standalone/conf/context.xml')
             .with_content(/<Resource name = "testdb"\n        auth = "Container"\n    \/>/) }
         end
+
+        context 'disable notifications' do
+          let(:params) {{
+            :version  => '6.3.4a',
+            :javahome => '/opt/java',
+          }}
+          it { should contain_file('/opt/jira/atlassian-jira-6.3.4a-standalone/bin/setenv.sh')
+            .with_content(Regexp.new('^#DISABLE_NOTIFICATIONS=')) }
```

Comment:
This regex will need to be updated. Something like /^DISABLE_NOTIFICATIONS=/. 

You don't need to use Regexp.new, You can just use /match/.